deploy/charts/cert-manager/templates/cainjector-poddisruptionbudget.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       app.kubernetes.io/instance: {{ .Release.Name }}
       app.kubernetes.io/component: "cainjector"
 
-  {{- with .Values.cainjector.podDisruptionBudget.minAvailable }}
+  {{- with and .Values.cainjector.podDisruptionBudget.minAvailable (not .Values.cainjector.podDisruptionBudget.maxUnavailable) }}
```

Comment:
I think this will cause the line below `minAvailable: {{ . }}` to render the wrong value (true/ false).

Reply:
You're right I'll fix this

Reply:
I tested the conditions locally now and they have the desired result.